src/Aeon/Aeon.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    </PropertyGroup>
+
```

Comment:
A new CSPROJ file means a new Nuget Package.
To differentiate it from the original Aeon emulator, could it be renamed Spice86.Aeon ?

Also, configure Source Link and such like this:

  <!-- Properties geared towards NuGet -->
  <PropertyGroup>
    <NoWarn>CS1591</NoWarn>
    <GenerateDocumentationFile>true</GenerateDocumentationFile>
    <version>3.0.18</version>
    <Authors>Kevin Ferrare, Maximilien Noal</Authors>
    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
    <Description>Reverse engineer and rewrite real mode dos programs</Description>
    <PackageTags>Reverse;Engineering;AvaloniaUI;GDB;Emulator;Cross;Platform</PackageTags>
    <PackageProjectUrl>https://github.com/OpenRakis/Spice86</PackageProjectUrl>
    <RepositoryUrl>https://github.com/OpenRakis/Spice86</RepositoryUrl>
    <RepositoryType>git</RepositoryType>
  </PropertyGroup>
  <!-- Source Link configuration -->
  <PropertyGroup>
    <PublishRepositoryUrl>true</PublishRepositoryUrl>
    <EmbedUntrackedSources>true</EmbedUntrackedSources>
    <ContinuousIntegrationBuild Condition="'$(GITHUB_ACTIONS)' == 'true'">true</ContinuousIntegrationBuild>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
    <PackageReference Include="DotNet.ReproducibleBuilds" Version="1.1.1">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
  </ItemGroup>

Reply:
Done. Feel free to change/add any properties to your likeing.